arroyo-compiler-service/src/main.rs

```diff
@@ -261,4 +263,54 @@ impl CompilerGrpc for CompileService {
                 .unwrap_or_else(|e| Status::internal(e.to_string()))
         })
     }
+
+    async fn check_udfs_mod(
+        &self,
+        request: Request<CheckUdfsModReq>,
+    ) -> Result<Response<CheckUdfsResp>, Status> {
+        // write udf to build_dir udfs module
+        let build_dir = &self.build_dir;
```

Comment:
this could use some logging so it's clear what the compiler service is doing (and timing information is also helpful)

Reply:
Sure, I've added some logging. Are you suggesting something like timing how long `cargo check` takes?

Reply:
Yes, like the existing timing information we have for compilation: https://github.com/ArroyoSystems/arroyo/blob/master/arroyo-compiler-service/src/main.rs#L208-L210

Reply:
Done